templates/vhost/vhost_ssl_header.erb

```diff
@@ -2,19 +2,19 @@
 server {
   <%- if @listen_ip.is_a?(Array) then -%>
     <%- @listen_ip.each do |ip| -%>
-  listen       <%= ip %>:<%= @ssl_port %> <% if @ssl_listen_option %>ssl<% end %><% if @spdy == 'on' %> spdy<% end %><% if @listen_options %> <%= @listen_options %><% end %>;
```

Comment:
Don't remove `ssl` or `spdy` flags here. They can be duplicated across listen directives.

This applies to the next 3 changed lines as well.